bus-mapping/src/evm/opcodes/begin_end_tx.rs

```diff
@@ -345,12 +345,7 @@ pub(crate) fn end_tx(
 
     // Write the next transaction id if we're not at the last tx
     if !state.tx_ctx.is_last_tx() {
```

Comment:
We can get rid of this if block since the body is empty now.
```suggestion
```

Reply:
🙆‍♂️ remove the whole block entirely